zone-service/src/main/java/org/cloudfoundry/identity/uaa/zone/OrchestratorZoneController.java

```diff
@@ -1,16 +1,17 @@
 package org.cloudfoundry.identity.uaa.zone;
 
 import static org.cloudfoundry.identity.uaa.zone.ErrorMessageUtil.MANDATORY_VALIDATION_MESSAGE;
-import static org.cloudfoundry.identity.uaa.zone.ErrorMessageUtil.getErrorMessagesHttpMessageNotReadable;
 import static org.cloudfoundry.identity.uaa.zone.ErrorMessageUtil.getErrorMessagesConstraintViolation;
+import static org.cloudfoundry.identity.uaa.zone.ErrorMessageUtil.getErrorMessagesHttpMessageNotReadable;
 import static org.cloudfoundry.identity.uaa.zone.ErrorMessageUtil.getErrorMessagesMethodArgumentInvalid;
+import static org.cloudfoundry.identity.uaa.zone.OrchestratorState.CREATE_IN_PROGRESS;
```

Comment:
Let's not import static values for OrchestratorState.  Importing static values for FOUND and NOT_FOUND might be confused with HttpStatus.FOUND and HttpStatus.NOT_FOUND.

FYI, I removed all static imports for OrchestratorState in the next commit.